subworkflows/local/run_read_coverage/main.nf

```diff
@@ -36,7 +36,7 @@ workflow RUN_READ_COVERAGE {
     // types
     //
 
-    if ( platform == "hifi" || platform == "clr" || platform == "ont" ) {
+    if ( params.reads_type == "hifi" || params.reads_type == "clr" || params.reads_type == "ont" ) {
```

Comment:
```suggestion
    if (params.reads_type in ["hifi", "clr", "ont"]) {
```

I see in the params schema that this doesn't have an enum - probably worth enforcing if you branch on specific conditions?

Reply:
That's a fair point. Done.